AdminServer/appscale/admin/instance_manager/instance.py

```diff
@@ -112,11 +126,17 @@ def create_java_start_cmd(app_name, port, load_balancer_host, max_heap,
     "--TQ_PROXY=" + options.tq_proxy,
     "--xmpp_path=" + options.load_balancer_ip,
     "--pidfile={}".format(pidfile),
-    "--external_api_port={}".format(api_server_port),
-    "--api_using_python_stub=app_identity_service",
-    os.path.dirname(web_inf_directory)
+    "--path_to_python_api_server={}".format(api_server)
   ]
 
+  for flag in api_server_flags:
+    cmd.append('--python_api_server_flag="{}"'.format(flag))
+
+  for api in apis_using_external_server:
+    cmd.append('--api_using_python_stub={}'.format(api))
```

Comment:
Should it be "{}"?

Reply:
None of the API service names have spaces, so I think it's safe to omit the quotes.